public/components/jobHistory.react.js

```diff
@@ -12,8 +12,21 @@ export default class JobHistory extends React.Component {
 
     render () {
         const jobHistoryNodes = this.props.data.map(jobHistory => {
+            const now = new Date().valueOf();
+            const started = new Date(jobHistory.startTime).valueOf();
+            const elapsedMins = (now - started) / 60000;
+            const yellowMins = 15;
+            const orangeMins = 20;
+            const rowClassName = jobHistory.status === "in progress"
```

Comment:
What's the motivation behind the colour coding? 

I'm a little concerned it could be misleading. For example, if we kick off a reindex that we expect to  take > 20 mins, what are we trying to imply by setting its status to orange after 20 mins has elapsed?

Reply:
It's 20 minutes since the last update.

Reply:
This should give us a good idea of a reindex that is taking a long time to start. Once running we should go back to green.

Reply:
I'm a bit confused re it being since the last update, since `elapsedMins = now - started`. I'd expect to see something like `elapsedMins = now - lastUpdated`. What am I missing?

Reply:
I see the confusion now. The colour coding from green to orange should apply to the in-progress indicator. I thought the history records got created immediately and updated as needed but that's not the case - they don't get created until after the reindex is finished/cancelled/failed etc. I'm looking at the progress indicator now, hoping I can transplant this particular code into that component, more or less.